src/library_async.js

```diff
@@ -383,7 +383,7 @@ mergeInto(LibraryManager.library, {
 #if ASYNCIFY_DEBUG
           dbg('ASYNCIFY: start unwind ' + Asyncify.currData);
 #endif
-          if (typeof Browser != 'undefined' && Browser.mainLoop.func) {
+          if (typeof Browser != 'undefined' && typeof Browser.mainLoop != 'undefined' && Browser.mainLoop.func) {
```

Comment:
I think you can just do `.. && Browser.mainLoop && Browser.mainLoop.func)` which is shorter.